Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# 1A-B:\n",
+    "n = 10\n",
+    "p = 0.5\n",
+    "# defining list of r values \n",
+    "r_values = list(range(n + 1)) \n",
+    "# list of pmf values \n",
+    "dist = [binom.pmf(r, n, p) for r in r_values ] \n",
+    "# plotting the graph\n",
+    "plt.bar(r_values, dist) \n",
```

Comment:
Part 2 Problem 1B:
- I'm not sure a bar chart is the most appropriate graph--it might imply an x-value can take any probability along the bar? A step function might be better to emphasize the discrete nature of both the x and probability values.